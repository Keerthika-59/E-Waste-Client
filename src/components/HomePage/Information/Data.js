export const data = [
    {
        'title': 'Waste Management',
        'text': 'There is no proper system for waste disposal. Sometimes for more than a week waste will not be collected. Waste management includes the activities and actions required  to manage waste from its inception to its final disposal. This includes the collection, transport, disposal of waste, together with monitoring and regulation of the waste management process and waste- related technologies, economic mechanisms.'
    },
    {
        'title': 'Disposing the Waste',
        'text': 'Some waste disposed be reused for some purpose. Unused waste will be disposed. Waste processing, and recycling or deposition of the waste materials of human society.  Waste materials are either liquid or solid in form, and their components may be either hazardous or inert that effects on health and the environment. Waste is typically applied to solid waste, sewage, hazardous waste, and electronic waste.'
    },
    {
        'title': 'Donating Old Items',
        'text': ' Unused good condition items such as old mobile phone, old Clothes, Food can be donated to needy. Donating provides an immediate benefit since the receiver can make use of the donated item within the shortest possible time.Leftover food originating from parties, weddings and restaurants will just go waste, unless it is donated in order to satiate the hunger of needy. '
    }
]