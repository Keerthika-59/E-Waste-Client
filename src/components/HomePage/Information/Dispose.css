.jumb-img {
    background-image: url('process.env.PUBLIC_URL/images/recycleRight_copy.jpeg');
}