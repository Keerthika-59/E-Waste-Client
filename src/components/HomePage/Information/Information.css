.text-large {
    font-size: larger;
    word-spacing: 3%;
    line-height: 2;
}

.heading {
    font-size: 40px;
    font-weight: 550;
}

.img {
    width: 230px;
    height: 230px;
}

.Info {
    display: flex;
    flex-direction: row;
}

@media (max-width: 800px) {
    .Info {
        flex-direction: column;
    }
    .heading {
        font-size: x-large;
        font-weight: 600;
    }
    .text-large {
        font-size: medium;
        word-spacing: 2%;
        line-height: 1.6;
    }
    .img {
        width: 130px;
        height: 130px;
        margin-left: 90px;
        margin-right: 50px;
    }
}