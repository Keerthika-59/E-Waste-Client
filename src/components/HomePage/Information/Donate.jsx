import {React,useEffect} from 'react';
import {Jumbotron,Media,Image} from "react-bootstrap"
import './Information.css'
import {data} from './Data.js';
const Donate = () => {
  useEffect(() => {
    window.scrollTo(0, 0)
  }, [])
    return (
        <div>
             <Jumbotron>
          <p className="text-center font-italic heading"> {data[2].title}</p>
  <Media as="li"  className="mt-5 Info">
    <Image
      className="mr-3 img" roundedCircle
      src={`${process.env.PUBLIC_URL}/assets/images/mediaDonate.jpg`}
     alt="Generic placeholder"
    />
    <Media.Body className="text-large text-justify ml-5 mr-5">
    <ul>
    <li>{data[2].text}</li>
          <li>{data[6].text}</li>
          <li>{data[7].text}</li>
          
      </ul>

    </Media.Body>
  </Media>
  </Jumbotron>
        </div>
    )
}

export default Donate
