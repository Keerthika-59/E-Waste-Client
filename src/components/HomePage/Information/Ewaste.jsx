import {React,useEffect} from 'react';
import {Jumbotron,Media,Image} from "react-bootstrap"
import './Information.css'
import {data} from './Data.js';
const Ewaste = () => {
  useEffect(() => {
    window.scrollTo(0, 0)
  }, [])
    return (
        <div>
          <section id="team" class="pb-5">
    <div class="container">
    <h5 class="section-title h1">Waste Management</h5>
    
        <div  class="text-white p-5 shadow-sm  rounded back">
          {/* <h3>DRIVEN BY OUR COMMITMENTS, GUIDED BY OUR VALUES</h3> */}
          <p class="lead">There is no proper system for waste disposal.  Sometimes for more than a week waste will not be collected. Waste management includes the activities and actions required  to manage waste from its inception to its final disposal.</p>
        </div>
        </div>
  </section>
            <Jumbotron>
      {/* <p className="text-center font-italic heading"> {data[0].title} </p>    */}
  <Media as="li" className="mt-5 Info text-justify">
    
    <Image
      className="mr-3 img" roundedCircle
      src={`${process.env.PUBLIC_URL}/assets/images/mediaWaste.jpg`}
      alt="Generic placeholder"
    />
    <Media.Body className="text-large ml-5 mr-5">
   {/* <h4> {data[4].name}</h4> */}
      <ul>
      <li>{data[0].text}</li>
          <li>{data[4].text}</li>
          <li>{data[5].text}</li>
      </ul>
    </Media.Body>
  </Media>
  </Jumbotron>
        </div>
    )
}

export default Ewaste
