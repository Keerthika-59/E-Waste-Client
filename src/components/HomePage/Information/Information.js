import React from "react";
import {Jumbotron,Media,Image} from "react-bootstrap"
import './Information.css'
import {data} from './Data.js';

const Information = () => {
  return (
    <>
     <ul className="list-unstyled mt-5">
      <Jumbotron>
      <p className="text-center font-italic heading"> {data[0].title} </p>   
  <Media as="li" className="mt-5 Info text-justify">
       
    <Image
      className="mr-3 img" roundedCircle
      src={`${process.env.PUBLIC_URL}/assets/images/mediaWaste.jpg`}
      alt="Generic placeholder"
    />
    <Media.Body className="text-large ml-5 mr-5">
      {data[0].text}

    </Media.Body>
  </Media>
  </Jumbotron>
  <Jumbotron>
          <p className="text-center font-italic heading"> {data[1].title}</p>

  <Media as="li" className="mt-5 Info text-justify">
    <Media.Body className="text-large ml-5 mr-5">
      {data[1].text}

    </Media.Body>
    <Image
      className="mr-3 img"roundedCircle
      src={`${process.env.PUBLIC_URL}/assets/images/mediaDispose.png`}
      alt="Generic placeholder"
    />
  </Media>
  </Jumbotron>
  <Jumbotron>
          <p className="text-center font-italic heading"> {data[2].title}</p>
  <Media as="li"  className="mt-5 Info">
    <Image
      className="mr-3 img" roundedCircle
      src={`${process.env.PUBLIC_URL}/assets/images/mediaDonate.jpg`}
     alt="Generic placeholder"
    />
    <Media.Body className="text-large text-justify ml-5 mr-5">
              {data[2].text}

    </Media.Body>
  </Media>
  </Jumbotron>
</ul> 
    </>
  );
};

export default Information;