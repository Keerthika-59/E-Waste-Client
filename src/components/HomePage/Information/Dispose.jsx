import React from 'react'
import {Jumbotron,Media,Image} from "react-bootstrap"
import {Button, Carousel, Container} from 'react-bootstrap';
import { Link } from "react-router-dom";
import './Information.css'
import './Dispose.css'
import {data} from './Data.js';
const Dispose = () => {
    return (
  //       <div>
  //           <Jumbotron>
  //         <p className="text-center font-italic heading"> {data[1].title}</p>

  // <Media as="li" className="mt-5 Info text-justify">
  //   <Media.Body className="text-large ml-5 mr-5">
  //   <ul>
  //         <li>{data[1].text}</li>
  //         <li>{data[3].text}</li>
  //         <li>{data[4].text}</li>
          
  //     </ul>

  //   </Media.Body>
  //   <Image
  //     className="mr-3 img"roundedCircle
  //     src={`${process.env.PUBLIC_URL}/assets/images/mediaDispose.png`}
  //     alt="Generic placeholder"
  //   />
  // </Media>
  // </Jumbotron>
  //       </div>

<div>
  {/* <Jumbotron style={{ 
  backgroundImage: `url(${process.env.PUBLIC_URL + '/images/recycleRight_copy.jpeg'})` 
}}>
          <p className="text-center font-italic heading"> {data[1].title}</p>

  <Media as="li" className="mt-5 Info text-justify">
    <Media.Body className="text-large ml-5 mr-5">
       <p>
    Disposing of waste will done properly only,
     when yu recycle it in a correct manner.
     We will help you out.
      </p>
    </Media.Body> */}
    {/* <Image
      className="mr-3 img"roundedCircle
      src={`${process.env.PUBLIC_URL}/assets/images/mediaDispose.png`}
      alt="Generic placeholder"
    /> 
  </Media>
  </Jumbotron>*/}
   <section id="team" class="pb-5">
    <div class="container">
   <h5 class="section-title h1">{data[1].title}</h5>
  <div className="text-center">
      <div class="col-lg-12">
        <div  class="text-white  p-5 shadow-sm  rounded back">
         
          <p class="lead">Disposing of waste will done properly only,
     when yu recycle it in a correct manner.
     We will help you out.
     Whether you’re at home or at work, we’ve got the tools
   to set you up for recycling success.</p>
      </div>
    </div>
    </div>

  <Jumbotron>
          <p className="text-center font-italic heading"> Recycle Right....</p>

  <Media as="li" className="mt-5 Info text-justify">
    <Media.Body className="text-large ml-5 mr-5">
    
      {/* <h1>Recycle Right....</h1> */}
    <ul>
      <li>
      We’re here to help you recycle right, answer common recycling questions and provide the information, 
      tools and resources you need to make recycling both simple and convenient.
      </li>
          <li> Whether you’re at home or at work, we’ve got the tools
   to set you up for recycling success. Get started with just
   six easy steps and visit our Resources for toolkits, labels, posters and more.</li>

      </ul>

    </Media.Body>
    <Image
      className="mr-3 img"roundedCircle
      src={`${process.env.PUBLIC_URL}/assets/images/mediaDispose.png`}
      alt="Generic placeholder"
    />
  </Media>
  </Jumbotron>
  <Carousel>
  <Carousel.Item interval={1000}>
  <img  className="d-block w-100"
        src={`${process.env.PUBLIC_URL}/images/DisposeCarousel/step1.jpg`}
        alt="Waste Management"
      />
    {/* <Carousel.Caption>
      <h3>First slide label</h3>
      <p>Nulla vitae elit libero, a pharetra augue mollis interdum.</p>
    </Carousel.Caption> */}
  </Carousel.Item>
  <Carousel.Item interval={500}>
  <img  className="d-block w-100"
        src={`${process.env.PUBLIC_URL}/images/DisposeCarousel/step2.jpg`}
        alt="Waste Management"
      />

    {/* <Carousel.Caption>
      <h3>Second slide label</h3>
      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
    </Carousel.Caption> */}
  </Carousel.Item>
  <Carousel.Item>
  <img  className="d-block w-100"
        src={`${process.env.PUBLIC_URL}/images/DisposeCarousel/step3.jpg`}
        alt="Waste Management"
    />
    {/* <Carousel.Caption>
      <h3>Third slide label</h3>
      <p>Praesent commodo cursus magna, vel scelerisque nisl consectetur.</p>
    </Carousel.Caption> */}
  </Carousel.Item>

  <Carousel.Item>
  <img  className="d-block w-100"
        src={`${process.env.PUBLIC_URL}/images/DisposeCarousel/step4.jpg`}
        alt="Waste Management"
    />
    {/* <Carousel.Caption>
      <h3>Third slide label</h3>
      <p>Praesent commodo cursus magna, vel scelerisque nisl consectetur.</p>
    </Carousel.Caption> */}
  </Carousel.Item>
  <Carousel.Item>
  <img  className="d-block w-100"
        src={`${process.env.PUBLIC_URL}/images/DisposeCarousel/step5.jpg`}
        alt="Waste Management"
    />
    {/* <Carousel.Caption>
      <h3>Third slide label</h3>
      <p>Praesent commodo cursus magna, vel scelerisque nisl consectetur.</p>
    </Carousel.Caption> */}
  </Carousel.Item>

  <Carousel.Item>
  <img  className="d-block w-100"
        src={`${process.env.PUBLIC_URL}/images/DisposeCarousel/step6.jpg`}
        alt="Waste Management"
    />
    {/* <Carousel.Caption>
      <h3>Third slide label</h3>
      <p>Praesent commodo cursus magna, vel scelerisque nisl consectetur.</p>
    </Carousel.Caption> */}
  </Carousel.Item>
  {/* <span aria-hidden="true" className="carousel-control-next-icon" /> */}
</Carousel>
</div>
</section>
</div>
     )
}

export default Dispose
