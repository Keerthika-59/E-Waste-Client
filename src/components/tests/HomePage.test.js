import React from 'react';
import { shallow } from 'enzyme';
import ReactDOM from 'react-dom';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
import Data from '../HomePage/Information/Data';
import Dispose from '../HomePage/Information/Dispose.jsx'
import Donate from '../HomePage/Information/Donate.jsx'
import Ewaste from '../HomePage/Information/Ewaste.jsx'
import Information from '../HomePage/Information/Information'

   test('renders test Dispose', () => {
    const div=document.createElement("div");
    <Route>
        ReactDom.render(<Dispose/>,div);
    </Route>
    ReactDOM.unmountComponentAtNode(div);
   });
   test('should test Dispose component', () => {
    const wrapper = shallow(<Dispose />);
    expect(wrapper).toMatchSnapshot();
   });

   test('renders test Donate', () => {
    const div=document.createElement("div");
    <Route>
        ReactDom.render(<Donate/>,div);
    </Route>
    ReactDOM.unmountComponentAtNode(div);
   });
   test('should test Donate component', () => {
    const wrapper = shallow(<Donate />);
    expect(wrapper).toMatchSnapshot();
   });

   test('renders test Ewaste', () => {
    const div=document.createElement("div");
    <Route>
        ReactDom.render(<Ewaste/>,div);
    </Route>
    ReactDOM.unmountComponentAtNode(div);
   });
   test('should test Ewaste component', () => {
    const wrapper = shallow(<Ewaste />);
    expect(wrapper).toMatchSnapshot();
   });

   test('renders test Information', () => {
    const div=document.createElement("div");
    <Route>
        ReactDom.render(<Information/>,div);
    </Route>
    ReactDOM.unmountComponentAtNode(div);
   });
   test('should test Information component', () => {
    const wrapper = shallow(<Information />);
    expect(wrapper).toMatchSnapshot();
   });
