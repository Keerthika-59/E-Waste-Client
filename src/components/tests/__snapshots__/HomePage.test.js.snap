// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Dispose component 1`] = `
<div>
  <section
    class="pb-5"
    id="team"
  >
    <div
      class="container"
    >
      <h5
        class="section-title h1"
      >
        Disposing the Waste
      </h5>
      <div
        className="text-center"
      >
        <div
          class="col-lg-12"
        >
          <div
            class="text-white  p-5 shadow-sm  rounded back"
          >
            <p
              class="lead"
            >
              Disposing of waste will don e properly only, when yu recycle it in a correct manner. We will help you out. Whether you’re at home or at work, we’ve got the tools to set you up for recycling success.
            </p>
          </div>
        </div>
      </div>
      <Jumbotron
        fluid={false}
      >
        <p
          className="text-center font-italic heading"
        >
           Recycle Right....
        </p>
        <Media
          as="li"
          className="mt-5 Info text-justify"
        >
          <MediaBody
            className="text-large ml-5 mr-5"
          >
            <ul>
              <li>
                We’re here to help you recycle right, answer common recycling questions and provide the information, tools and resources you need to make recycling both simple and convenient.
              </li>
              <li>
                 Whether you’re at home or at work, we’ve got the tools to set you up for recycling success. Get started with just six easy steps and visit our Resources for toolkits, labels, posters and more.
              </li>
            </ul>
          </MediaBody>
          <Image
            alt="Generic placeholder"
            className="mr-3 img"
            fluid={false}
            rounded={false}
            roundedCircle={true}
            src="/assets/images/mediaDispose.png"
            thumbnail={false}
          />
        </Media>
      </Jumbotron>
      <Carousel
        controls={true}
        defaultActiveIndex={0}
        fade={false}
        indicators={true}
        interval={5000}
        keyboard={true}
        nextIcon={
          <span
            aria-hidden="true"
            className="carousel-control-next-icon"
          />
        }
        nextLabel="Next"
        pause="hover"
        prevIcon={
          <span
            aria-hidden="true"
            className="carousel-control-prev-icon"
          />
        }
        prevLabel="Previous"
        slide={true}
        touch={true}
        wrap={true}
      >
        <CarouselItem
          interval={1000}
        >
          <img
            alt="Waste Management"
            className="d-block w-100"
            src="/images/DisposeCarousel/step1.jpg"
          />
        </CarouselItem>
        <CarouselItem
          interval={500}
        >
          <img
            alt="Waste Management"
            className="d-block w-100"
            src="/images/DisposeCarousel/step2.jpg"
          />
        </CarouselItem>
        <CarouselItem>
          <img
            alt="Waste Management"
            className="d-block w-100"
            src="/images/DisposeCarousel/step3.jpg"
          />
        </CarouselItem>
        <CarouselItem>
          <img
            alt="Waste Management"
            className="d-block w-100"
            src="/images/DisposeCarousel/step4.jpg"
          />
        </CarouselItem>
        <CarouselItem>
          <img
            alt="Waste Management"
            className="d-block w-100"
            src="/images/DisposeCarousel/step5.jpg"
          />
        </CarouselItem>
        <CarouselItem>
          <img
            alt="Waste Management"
            className="d-block w-100"
            src="/images/DisposeCarousel/step6.jpg"
          />
        </CarouselItem>
      </Carousel>
    </div>
  </section>
</div>
`;

exports[`should test Donate component 1`] = `
<div>
  <section
    class="pb-5"
    id="team"
  >
    <div
      class="container"
    >
      <h5
        class="section-title h1"
      >
        Donate to Needy
      </h5>
      <div
        class="text-white p-5 shadow-sm  rounded back"
      >
        <p
          class="lead"
        >
          Unused good condition items such as old mobile phone, old Clothes, Food can be donated to needy. Donating provides an immediate benefit since the receiver can make use of the donated item within the shortest possible time.
        </p>
      </div>
    </div>
  </section>
  <Jumbotron
    fluid={false}
  >
    <Media
      as="li"
      className="mt-5 Info"
    >
      <Image
        alt="Generic placeholder"
        className="mr-3 img"
        fluid={false}
        rounded={false}
        roundedCircle={true}
        src="/assets/images/mediaDonate.jpg"
        thumbnail={false}
      />
      <MediaBody
        className="text-large text-justify ml-5 mr-5"
      >
        <ul>
          <li>
            Unused good condition items such as old mobile phone, old Clothes, Food can be donated to needy. Donating provides an immediate benefit since the receiver can make use of the donated item within the shortest possible time. Leftover food originating from parties, weddings and restaurants will just go waste, unless it is donated in order to satiate the hunger of needy.
          </li>
          <li>
            Dispose of leftover food in a responsible manner.
          </li>
          <li>
            Donate any usable item to the needy.
          </li>
        </ul>
      </MediaBody>
    </Media>
  </Jumbotron>
</div>
`;

exports[`should test Ewaste component 1`] = `
<div>
  <section
    class="pb-5"
    id="team"
  >
    <div
      class="container"
    >
      <h5
        class="section-title h1"
      >
        Waste Management
      </h5>
      <div
        class="text-white p-5 shadow-sm  rounded back"
      >
        <p
          class="lead"
        >
          There is no proper system for waste disposal.  Sometimes for more than a week waste will not be collected. Waste management includes the activities and actions required  to manage waste from its inception to its final disposal.
        </p>
      </div>
    </div>
  </section>
  <Jumbotron
    fluid={false}
  >
    <Media
      as="li"
      className="mt-5 Info text-justify"
    >
      <Image
        alt="Generic placeholder"
        className="mr-3 img"
        fluid={false}
        rounded={false}
        roundedCircle={true}
        src="/assets/images/mediaWaste.jpg"
        thumbnail={false}
      />
      <MediaBody
        className="text-large ml-5 mr-5"
      >
        <ul>
          <li>
            There is no proper system for waste disposal.  Sometimes for more than a week waste will not be collected. Waste management includes the activities and actions required  to manage waste from its inception to its final disposal. This includes the collection, transport, disposal of waste, together with monitoring and regulation of the waste management process and waste- related technologies, economic mechanisms.
          </li>
          <li>
            With Waste Management, recycling goes beyond just bottles, cans and paper. Some things we are used to throwing away can often be safely recycled too, from old TV remotes to batteries to compact fluorescent bulbs and more.
          </li>
          <li>
            Due to safety concerns, not all items can be collected with your normal trash pickup and hazardous materials, if not properly disposed of, may have an adverse impact on the environment.
          </li>
        </ul>
      </MediaBody>
    </Media>
  </Jumbotron>
</div>
`;

exports[`should test Information component 1`] = `
<Fragment>
  <br />
  <br />
  <br />
  <div
    class="container"
  >
    <div
      class="row"
    >
      <div
        class="col-md-3 col-sm-6"
      >
        <div
          class="counter"
        >
          <div
            class="counter-icon"
          >
            <i
              class="fa fa-globe"
            />
          </div>
          <h3>
            Happy clients
          </h3>
          <span
            class="counter-value"
          >
            565
          </span>
        </div>
      </div>
      <div
        class="col-md-3 col-sm-6"
      >
        <div
          class="counter orange"
        >
          <div
            class="counter-icon"
          >
            <i
              class="fa fa-rocket"
            />
          </div>
          <h3>
            Volunteers
          </h3>
          <span
            class="counter-value"
          >
            20
          </span>
        </div>
      </div>
      <div
        class="col-md-3 col-sm-6"
      >
        <div
          class="counter"
        >
          <div
            class="counter-icon"
          >
            <i
              class="fa fa-globe"
            />
          </div>
          <h3>
            Donation
          </h3>
          <span
            class="counter-value"
          >
            65
          </span>
        </div>
      </div>
      <div
        class="col-md-3 col-sm-6"
      >
        <div
          class="counter orange"
        >
          <div
            class="counter-icon"
          >
            <i
              class="fa fa-rocket"
            />
          </div>
          <h3>
            Projects
          </h3>
          <span
            class="counter-value"
          >
            87
          </span>
        </div>
      </div>
    </div>
  </div>
  <ul
    className="list-unstyled mt-5"
  >
    <Jumbotron
      fluid={false}
    >
      <p
        className="text-center font-italic heading"
      >
         
        Waste Management
         
      </p>
      <Media
        as="li"
        className="mt-5 Info text-justify"
      >
        <Image
          alt="Generic placeholder"
          className="mr-3 img"
          fluid={false}
          rounded={false}
          roundedCircle={true}
          src="/assets/images/mediaWaste.jpg"
          thumbnail={false}
        />
        <MediaBody
          className="text-large ml-5 mr-5"
        >
          There is no proper system for waste disposal.  Sometimes for more than a week waste will not be collected. Waste management includes the activities and actions required  to manage waste from its inception to its final disposal. This includes the collection, transport, disposal of waste, together with monitoring and regulation of the waste management process and waste- related technologies, economic mechanisms.
          <Button
            active={false}
            disabled={false}
            variant="link"
          >
            <Link
              style={
                Object {
                  "textDecoration": "none",
                }
              }
              to="/Ewaste"
            >
              Learn More...
            </Link>
          </Button>
        </MediaBody>
      </Media>
    </Jumbotron>
    <Jumbotron
      fluid={false}
    >
      <p
        className="text-center font-italic heading"
      >
         
        Disposing the Waste
      </p>
      <Media
        as="li"
        className="mt-5 Info text-justify"
      >
        <MediaBody
          className="text-large ml-5 mr-5"
        >
          Some waste disposed be reused for some purpose. Unused waste will be disposed. Waste processing, and recycling or deposition of the waste materials of human society.  Waste materials are either liquid or solid in form, and their components may be either hazardous or inert that effects on health and the environment. Waste is typically applied to solid waste, sewage, hazardous waste, and electronic waste.
          <Button
            active={false}
            disabled={false}
            variant="link"
          >
            <Link
              style={
                Object {
                  "textDecoration": "none",
                }
              }
              to="/Disposing"
            >
              Learn More...
            </Link>
          </Button>
        </MediaBody>
        <Image
          alt="Generic placeholder"
          className="mr-3 img"
          fluid={false}
          rounded={false}
          roundedCircle={true}
          src="/assets/images/mediaDispose.png"
          thumbnail={false}
        />
      </Media>
    </Jumbotron>
    <Jumbotron
      fluid={false}
    >
      <p
        className="text-center font-italic heading"
      >
         
        Donating Old Items
      </p>
      <Media
        as="li"
        className="mt-5 Info"
      >
        <Image
          alt="Generic placeholder"
          className="mr-3 img"
          fluid={false}
          rounded={false}
          roundedCircle={true}
          src="/assets/images/mediaDonate.jpg"
          thumbnail={false}
        />
        <MediaBody
          className="text-large text-justify ml-5 mr-5"
        >
          Unused good condition items such as old mobile phone, old Clothes, Food can be donated to needy. Donating provides an immediate benefit since the receiver can make use of the donated item within the shortest possible time. Leftover food originating from parties, weddings and restaurants will just go waste, unless it is donated in order to satiate the hunger of needy.
          <Button
            active={false}
            disabled={false}
            variant="link"
          >
            <Link
              style={
                Object {
                  "textDecoration": "none",
                }
              }
              to="/Donating"
            >
              Learn More...
            </Link>
          </Button>
        </MediaBody>
      </Media>
    </Jumbotron>
  </ul>
</Fragment>
`;
